src/pages/RepoPage/BundlesTab/BundleOnboarding/BundleOnboarding.tsx

```diff
@@ -55,14 +73,27 @@ interface BundlerSelectorProps {
 function BundlerSelector({ provider, owner, repo }: BundlerSelectorProps) {
   const location = useLocation()
   const history = useHistory()
-  const { bundleOnboarding, bundleWebpackOnboarding, bundleRollupOnboarding } =
-    useNavLinks()
+  const {
+    bundleNuxtOnboarding,
+    bundleOnboarding,
+    bundleRemixOnboarding,
+    bundleRollupOnboarding,
+    bundleSolidStartOnboarding,
+    bundleSvelteKitOnboarding,
+    bundleWebpackOnboarding,
+  } = useNavLinks()
   const urls = {
     Vite: bundleOnboarding.path({ provider, owner, repo }),
     Rollup: bundleRollupOnboarding.path({ provider, owner, repo }),
     Webpack: bundleWebpackOnboarding.path({ provider, owner, repo }),
+    Remix: bundleRemixOnboarding.path({ provider, owner, repo }),
+    Nuxt: bundleNuxtOnboarding.path({ provider, owner, repo }),
+    SvelteKit: bundleSvelteKitOnboarding.path({ provider, owner, repo }),
+    SolidStart: bundleSolidStartOnboarding.path({ provider, owner, repo }),
   }
 
+  const itemClass = 'flex-none w-[calc((100%-32px)/3)]'
```

Comment:
This changes up the UI a little bit because having this many tile items with the current flex styling would overflow horizontally. Each item would have a minimum width of the widest single word.

After these changes, for this specific instance in BundleOnboarding, the tiles will wrap with max of 3 per row and not fully flex expand like they normally do.